spec/operators/windowTime-spec.js

```diff
@@ -123,6 +131,37 @@ describe('Observable.prototype.windowTime', function () {
     expectObservable(result, unsub).toBe(expected, values);
   });
 
+  it('should dispose window Subjects if the outer is unsubscribed early', function (done) {
+    var virtualTimeScheduler = new Rx.VirtualTimeScheduler();
```

Comment:
So, my standard approach was initially marble tests of course. Then I got into problems where I wanted to subscribe to a window (an inner Observable of the outer result) **AFTER** the whole outer is unsubscribed manually. I didn't figure out how to do that with marble diagrams without creating additional weirdness around it. I can give it another try. (the same applies to all other windowVariation operator tests).